server/index.js

```diff
@@ -69,20 +69,54 @@ app.get('/user/find', (req, res) => {
 	});
 });
 
+const matchMaker = () => {
+	setInterval(() => {
```

Comment:
Why not run a function whenever a new user joins instead of using setinterval

Reply:
That makes sense. I will do it now.

Reply:
Fixed this issue.